src/devices/cpu/t11/t11.cpp

```diff
@@ -370,6 +417,25 @@ void t11_device::state_string_export(const device_state_entry &entry, std::strin
 	}
 }
 
+void k1801vm1_device::state_string_export(const device_state_entry &entry, std::string &str) const
+{
+	switch (entry.index())
+	{
+		case STATE_GENFLAGS:
+			str = string_format("%c%c%c%c%c%c%c%c",
+				m_psw.b.l & 0x80 ? 'I':'.',
+				m_psw.b.l & 0x40 ? '?':'.',
+				m_psw.b.l & 0x20 ? '?':'.',
+				m_psw.b.l & 0x10 ? 'T':'.',
+				m_psw.b.l & 0x08 ? 'N':'.',
+				m_psw.b.l & 0x04 ? 'Z':'.',
+				m_psw.b.l & 0x02 ? 'V':'.',
+				m_psw.b.l & 0x01 ? 'C':'.'
+			);
+			break;
+	}
```

Comment:
It’s generally good practice to call the inherited member function if it isn’t something you’re handling yourself (I know that won’t do anything here).